containers/environment.yml

```diff
@@ -0,0 +1,12 @@
+name: exoseq
```

Comment:
Final minor comment sorry - could this have `nf-core` in the environment name so that it's clearer for the user? Maybe `nf-core-exoseq`?